jupyter_ydoc/ynotebook.py

```diff
@@ -98,8 +100,8 @@ def get_cell(self, index: int) -> Dict[str, Any]:
         :return: A cell.
         :rtype: Dict[str, Any]
         """
-        meta = json.loads(self._ymeta.to_json())
-        cell = json.loads(self._ycells[index].to_json())
+        meta = json.loads(str(self._ymeta))
```

Comment:
I'm a bit worry about that API. if `__str__` is returning the serialized JSON value, we cannot distinguish between a _Text_ containing a serialized JSON and a _Map_ or an _Array_.

So this contradicts with the [goal](https://docs.python.org/3/reference/datamodel.html?highlight=__str__#object.__str__) of `__str__`:

> Called by [str(object)](https://docs.python.org/3/library/stdtypes.html#str) and the built-in functions [format()](https://docs.python.org/3/library/functions.html#format) and [print()](https://docs.python.org/3/library/functions.html#print) to compute the “informal” or nicely printable [string](https://docs.python.org/3/library/stdtypes.html#textseq) representation of an object. The return value must be a string object.
> 
> This method differs from [object.__repr__()](https://docs.python.org/3/reference/datamodel.html?highlight=__str__#object.__repr__) in that there is no expectation that [__str__()](https://docs.python.org/3/reference/datamodel.html?highlight=__str__#object.__str__) return a valid Python expression: a more convenient or concise representation can be used.

Reply:
On the other hand, this is aligned with the Python builtin counterparts:
- `Text` -> `str`:
```py
v = "foo"
str(v)
# "foo"
```
- `Array` -> `list`:
```py
v = [1, 2]
str(v)
# "[1, 2]"
```
- `Map` -> `dict`:
```py
v = {"foo": 1, "bar": 2}
str(v)
# "{'foo': 1, 'bar': 2}"
```
But I'm open to suggestions. In Ypy, I think that `to_json()` is doing the [same thing](https://github.com/y-crdt/ypy/pull/102).

Reply:
Yeah, the usage of str is a bit slippery.